Controllers/Patch-reviews.js

```diff
@@ -0,0 +1,14 @@
+const { patchReviewsModel } = require("../Models/Patch-reviews-model")
+
+
+exports.patchReviews = ((req, res, next) => {
+    const reviewID = req.params.review_id
+    const voteChange = req.body.vote
+
+    patchReviewsModel(reviewID, voteChange).then((review) => {
+        res.status(200).send(review.rows[0])
```

Comment:
What's being sent in .send() isn't correct - it needs to be an object with the key of review... you wouldn't usually send back an index of an array directly or send back directly what the db.query returns from the model, you would deal with the array returned in the model, see next comment.